plugin.d.ts

```diff
@@ -4,6 +4,10 @@ declare module 'fastify' {
   interface FastifyRequest {
     rawBody?: string | Buffer
   }
+
+  interface FastifyContextConfig {
+    rawBody?: boolean
+  }
```

Comment:
why updating a plugin and not the server?
I mean, there are some security releases too since 4.7.0

Anyway, we should bump the version here:

https://github.com/Eomm/fastify-raw-body/blob/main/plugin.js#L106